src/components/OSMElementData/OSMElementData.scss

```diff
@@ -0,0 +1,17 @@
+@import '../../variables.scss';
+
+dl.tag-list {
```

Comment:
I first inspired myself from `KeyboardShortcutList.scss` but it did not really fit due to too lob tag keys. I think using a grid here is better too. I would happily change the design to something else if we already have an UI for tags or if there is a better UI concept.

Reply:
this implementation looks good to me